frontend/packages/console-app/src/components/cloud-shell/AutoResizeTerminal.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react';
+import { Terminal as XTerminal } from 'xterm';
+import Measure from 'react-measure';
+import * as fit from 'xterm/lib/addons/fit/fit';
+import * as full from 'xterm/lib/addons/fullscreen/fullscreen';
+
+XTerminal.applyAddon(fit);
+XTerminal.applyAddon(full);
+
+type TerminalProps = {
+  onData: any;
```

Comment:
Don't use `any` type.
Is this a string?

Reply:
update to function with string arg and void return